dotenv-sample

```diff
@@ -21,13 +21,9 @@ AIRLOCK_API_TOKEN=
 AIRLOCK_API_ENDPOINT="https://localhost:9000/api/v2"
 
 # Opentelemetry
-export OTEL_EXPORTER_OTLP_ENDPOINT="https://api.honeycomb.io"
```

Comment:
wdyt about keeping a line for `OTEL_EXPORTER_OTLP_ENDPOINT` here but with a note that it's provided by the default config in backend-server? feel like that's would document that the app requires it but our deployment provides it automatically? make it slightly more legible for someone unfamiliar with the app?

Reply:
IMO, dotenv-sample is for setting up dev envs.

I don't see it as a skeleton/documentation for production config, just the minimal/useful set of things you need to run locally and may want to tweak.  It has no relation to the config in production.

Now, we may want something like that for prod, but I think dotenv-sample should be optimised for local dev use, not full belts and braces production config

Reply:
fair enough

Reply:
I guess I do see `dotenv-sample` as skeleton/documentation for any config prod or dev, although in practice once we've set up the deployment once we're not going to look at it again so it's not a big deal. Seems more convenient to me in general to maintain the list of all config options in there rather than somewhere else (a markdown doc file?), but we also don't really need to document every little thing right now.